kolibri/content/utils/sqlalchemybridge.py

```diff
@@ -152,17 +157,32 @@ def get_default_db_string():
             dbname=destination_db['NAME'],
         )
 
+class SchemaNotFoundError(Exception):
+    pass
+
 class Bridge(object):
 
     def __init__(self, sqlite_file_path=None, app_name=None):
         if sqlite_file_path is None:
+            # If sqlite_file_path is None, we are referencing the Django default database
             self.connection_string = get_default_db_string()
+            self.Base = BASES[CONTENT_SCHEMA_VERSION]
         else:
+            # Otherwise, we are accessing an external content database.
+            # So we try each of our historical database schema in order to see
+            # which glass slipper fits! If none do, just turn into a pumpkin.
             self.connection_string = sqlite_connection_string(sqlite_file_path)
+            for version in CONTENT_DB_SCHEMA_VERSIONS:
+                self.Base = BASES[version]
+                self.session, self.engine = make_session(self.connection_string)
+                if (db_matches_schema(self.Base, self.session)):
+                    break
+            else:
+                raise SchemaNotFoundError('No matching schema found for this database')
```

Comment:
Just raising an error here if we can't find a schema that matches, at the moment this is not caught anywhere, and will just bubble up until it raises an error in the import channel command.

Reply:
(This seems like an unlikely event, unless someone has a corrupted database, or is just trying to stick any old database into the machinery).